src/inputs/input/input.jsx

```diff
@@ -246,15 +348,17 @@ const styles = (theme) => ({
     },
 });
 
+/**
+ * The Input is used for gathering single line data from the user.
+ */
 class Input extends React.PureComponent {
     constructor(props) {
         super(props);
 
         this.state = {
+            hasValue: !!props.value,
             isFocused: false,
-            inputActionsTopPosition: 0,
-            showRequiredIndicator: props.required,
-            // value: props.value || props.value === 0 ? props.value : ''
+            requirementFullfillment: null,
```

Comment:
"Fulfillment" has one too many "l"s.  Should be `requirementFulfillment`.

Reply:
Fixed.

Reply:
Looks good.  Resolving.